docs/reference/commandline/plugin_ls.md

```diff
@@ -24,8 +24,10 @@ Aliases:
   ls, list
 
 Options:
-      --help	   Print usage
-      --no-trunc   Don't truncate output
+      --format string   Pretty-print plugins using a Go template
```

Comment:
Lets also add bash completion alongside. You can pick my commit. https://github.com/docker/docker/commit/1c8d0c13b0bda17c17ab7348a9c69ff789c81240